website/pages/en/network/developing.mdx

```diff
@@ -2,52 +2,104 @@
 
-Some developers will engage with the full subgraph lifecycle on the network, publishing, querying and iterating on their own subgraphs. Some may be focused on subgraph development, building open APIs which others can build on. Some may be application focused, querying subgraphs deployed by others.
+Once defined, a subgraph can be deployed to [Subgraph Studio](/deploying/deploying-a-subgraph-to-studio/). In Subgraph Studio, you can do the following:
 
-### Developers and Network Economics
+- Use its staging environment to index the deployed subgraph and make it available for review.
+- Verify that your subgraph doesn't have any indexing errors and works as expected.
 
-Developers are a key economic actor in the network, locking up GRT in order to encourage indexing, and crucially querying subgraphs, which is the network's primary value exchange. Subgraph developers also burn GRT whenever a subgraph is updated.
+## Publish to the Network
+
+When you are happy with your subgraph, you can [publish it](/publishing/publishing-a-subgraph/) to The Graph Network.
+
+- To put your subgraph into production and ensure indexers can index it, you must publish it to The Graph Network.
+  - This is an on-chain action, which registers the subgraph and makes it discoverable by Indexers.
+- Published subgraphs have a corresponding NFT, which is then easily transferable.
+- Published subgraphs have associated metadata, which provides other network participants with useful context and information.
+
+## Add Signal for Indexing
+
+Published subgraphs are unlikely to be picked up by Indexers without signal. To encourage indexing you should add signal to your subgraph. Learn more about signaling and [curating](/network/curating/) on The Graph.
+
+### What is signal?
+
+- Signal is locked GRT associated with a given subgraph. It indicates to indexers that a given subgraph will receive query volume and it contributes to the indexing rewards available for processing it.
+- Third party Curators may also signal on a given subgraph, if they deem the subgraph likely to drive query volume.
+
+## Querying & Application Development
+
+Once a subgraph has been processed by Indexers and is ready to [query](/querying/querying-the-graph/), developers can start using their subgraph in their dapps.
+
+- When a query is made, The Graph Network selects an Indexer who has processed the subgraph to serve that query. Query fees are paid in GRT.
+- Developers must generate an API key and fund it with GRT to make queries, which can be done in [Subgraph Studio](https://thegraph.com/studio/).
+- Developers can set a maximum query fee, in order to control their costs, and limit their API key to a given subgraph or origin domain.
+  - Subgraph Studio provides API key usage data.
+- Developers can express an Indexer preference to the gateway, for example preferring Indexers whose query response is faster, or whose data is most up to date. These controls are set in Subgraph Studio.
```

Comment:
Are you sure about this as well? I know it used to be the case, but we've at least talked about removing this.